tools/helper.py

```diff
@@ -0,0 +1,3 @@
+def chunk_list(lst, n):
+    for i in range(0, len(lst), n):
+        yield lst[i:i + n]
```

Comment:
I think it make sense that we include the YamlRead.py also in helper.py. So that we have only one helper file.